spec/components/schemas/content/instagram/optin_response.ts

```diff
@@ -0,0 +1,33 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from '../base';
+import { createComponentRef } from '../../../../../utils/ref';
+
+const optInResponse: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'string',
+        example: 'optin_response',
+      },
+      notificationRecipient: {
+        type: 'string',
+        description: 'A token that you will be using to send recurring messages for a specific topic that user has opted-in. You will receive a unique token for each topic user opts-in. This attribute must be sent as `TO` attribute when sending a recurring notification message.',
```

Comment:
Acho melhor esquecer a palavra token já que o único conceito de token nessa API é o token de autenticação.Uma ideia de refatoração desse texto:
```
The contact id that needs to be used to send the recurring messages for the topic the user has opted-in. You will receive a unique contact id for each topic the user opts-in. Therefore, this needs to be used the recipient (`to` attribute) when sending a recurring message. This contact id has an expiration timestamp.
```

Seguindo nessa linha de nomenclatura (onde na real não tá sendo usado notification e sim topic), em vez da minha sugestão inicial de chamar esse campo de `notificationRecipient`, que tal chamar de `topicRecipient`?

Reply:
Opa. Eu pensei em usar o `topic` pra quando for falar do atributo `TITLE`.
Pq é ele que vai ditar se o "tópico" da mensagem é repetido ou não.
_(Regra de que não pode repetir o tópico na hora de enviar um optin_request)_

que tu acha? Usamos tópic para o token mesmo e utilizamos outro nome para o Title?

Reply:
É que nesse contexto eu vejo topic mais próximo de subscription. Por isso `topicRecipient`, e o title poderia até ser `topicTitle`. Tipo, o conceito de notification achei que ficou meio perdido aqui.

Outra opção, de repente, se for mesmo focar em message os conceitos, poderia ser `messageRecipient`, ou ainda, apesar de meio longo, fica bem claro, `recurringMessageRecipient`.

OBS: E se o title tem que ser único, era legal incluir isso também na descrição do campo no optin_request.